core/src/test/java/com/linecorp/armeria/common/QueryParamsBaseTest.java

```diff
@@ -116,6 +116,27 @@ void testGetBooleanOperation() {
         params.add("baz", "false");
         assertThat(params.containsBoolean("baz", false)).isTrue();
         assertThat(params.containsBoolean("baz", true)).isFalse();
+
+        params.add("dup1", "v1");
+        params.add("dup1", "v2");
+        params.add("dup1", "true");
+        params.add("dup1", "false");
+        assertThat(params.containsBoolean("dup1", true)).isTrue();
+        assertThat(params.containsBoolean("dup1", false)).isTrue();
+
+        params.add("dup2", "v1");
+        params.add("dup2", "v2");
+        params.add("dup2", "1");
+        params.add("dup2", "0");
+        assertThat(params.containsBoolean("dup2", true)).isTrue();
+        assertThat(params.containsBoolean("dup2", false)).isTrue();
```

Comment:
```java
        headers.add("dup1", "v1");
        headers.add("dup1", "v2");
        headers.add("dup1", "true");

// Now: containsBoolean -> getBoolean("dup1") -> "v1" -> contains: false
        assertThat(headers.containsBoolean("dup1", true)).isTrue(); 
```
With current `containsBoolean`, above test case failed cause it uses `getBoolean` inside!